supabase_auth/types.py

```diff
@@ -19,48 +25,57 @@
 
 from typing_extensions import Literal, NotRequired, TypedDict
 
-Provider = Literal[
-    "apple",
-    "azure",
-    "bitbucket",
-    "discord",
-    "facebook",
-    "figma",
-    "fly",
-    "github",
-    "gitlab",
-    "google",
-    "kakao",
-    "keycloak",
-    "linkedin",
-    "linkedin_oidc",
-    "notion",
-    "slack",
-    "slack_oidc",
-    "spotify",
-    "twitch",
-    "twitter",
-    "workos",
-    "zoom",
-]
 
-EmailOtpType = Literal[
-    "signup", "invite", "magiclink", "recovery", "email_change", "email"
-]
+class Provider(StrEnum):
+    Apple = "apple"
+    Azure = "azure"
+    Bitbucket = "bitbucket"
+    Discord = "discord"
+    Facebook = "facebook"
+    Figman = "figma"
+    Fly = "fly"
+    Github = "github"
+    Gitlab = "gitlab"
+    Google = "google"
+    Kakao = "kakao"
+    Keycloak = "keycloak"
+    Linkedin = "linkedin"
```

Comment:
Can drop Linkedin and Slack as they have been replaced by `slack_oidc` and `linkedin_oidc`

Reply:
ignore, apologies. we keep them on the js lib so let's keep them here too for now.

Reply:
Ok, I thought it was for backwards compat.